### PR TITLE
Update missing Chinese translations of v2.8 and latest versions.

### PR DESCRIPTION
**Pull request by STARRY-S**
## Description

Update Chinese translations of the `latest` and `v2.8` versions.

Modified files of version `latest` & `v2.8`:
1. `reference-guides/best-practices/best-practices.md`
2. `reference-guides/best-practices/rancher-server/rancher-server.md`
3. `reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md`
4. `reference-guides/best-practices/rancher-managed-clusters/rancher-managed-clusters.md`
5. `reference-guides/rancher-manager-architecture/rancher-manager-architecture.md`
6. `reference-guides/cluster-configuration/cluster-configuration.md`
7. `reference-guides/cluster-configuration/rancher-server-configuration/rancher-server-configuration.md`
8. `reference-guides/cluster-configuration/rancher-server-configuration/gke-cluster-configuration/gke-cluster-configuration.md`
9. `reference-guides/cluster-configuration/rancher-server-configuration/use-existing-nodes/use-existing-nodes.md`
10. `reference-guides/cluster-configuration/downstream-cluster-configuration/downstream-cluster-configuration.md`
11. `reference-guides/cluster-configuration/downstream-cluster-configuration/node-template-configuration/node-template-configuration.md`
12. `reference-guides/cluster-configuration/downstream-cluster-configuration/machine-configuration/machine-configuration.md`
13. `reference-guides/single-node-rancher-in-docker/single-node-rancher-in-docker.md`
14. `reference-guides/backup-restore-configuration/backup-restore-configuration.md`
15. `how-to-guides/advanced-user-guides/enable-api-audit-log-in-downstream-clusters.md`

### CODE REVIEW

**Comment by STARRY-S** [comment]
@JacieChao Hi, could you please help me to review this PR? Thanks!

File: i18n/zh/docusaurus-plugin-content-docs/current/reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+你应当删除 Cluster Manager UI 中遗留的所有旧应用程序，并将其替换为 Cluster Explorer UI 中的应用程序。只在 Cluster Explorer UI 中创建任何新应用程序。
+
+### 使用授权集群端点 (ACE)
+
+[授权集群端点 Authorized Cluster Endpoint](../../../reference-guides/rancher-manager-architecture/communicating-with-downstream-user-clusters.md#4-授权集群端点) (ACE) 提供了 Rancher 部署的 RKE、RKE2 和 K3s 下游集群的直接 Kubernetes API 访问。启用后，ACE 会为生成的 `kubeconfig` 文件配置直接访问下游集群 Endpoint，从而绕过 Rancher 代理。在可以直接访问下游集群 Kubernetes API 的场景下，可以减少 Rancher 负载。有关更多信息，请参阅[授权集群端点](../../../reference-guides/rancher-manager-architecture/communicating-with-downstream-user-clusters.md#4-授权集群端点)配置说明。
+
+### 减少事件处理程序执行
+
+Rancher 的大部分逻辑发生在 Handler 事件处理程序。每当资源对象产生更新或 Rancher 启动时，这些 Handler 会执行。除此之外，它们会每隔 15 小时在 Rancher 计划的同步缓存时再运行一次，这可能会导致 Rancher 运行过程过程中出现大量性能消耗。可使用 `CATTLE_SYNC_ONLY_CHANGED_OBJECTS` 环境变量禁用计划的 Handler 处理程序执行。如果每 15 小时出现一次性能资源高峰，此设置会有所帮助。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md:81]: - Handler 事件处理程序 => Event Handler 上
- 这些 Handler 会执行 => 这些资源对应的 Event Handler 都会执行
- Rancher 运行过程过程 => Rancher 运行过程

File: i18n/zh/docusaurus-plugin-content-docs/current/reference-guides/cluster-configuration/rancher-server-configuration/use-existing-nodes/use-existing-nodes.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+```
+Key=kubernetes.io/cluster/CLUSTERID, Value=shared
+```
+
+## 可选的后续步骤
+
+创建集群后，你可以通过 Rancher UI 访问集群。最佳实践建议你设置以下访问集群的备用方式：
+
+- **通过 kubectl CLI 访问你的集群**：按照[这些步骤](../../../../how-to-guides/new-user-guides/manage-clusters/access-clusters/use-kubectl-and-kubeconfig.md#在工作站使用-kubectl-访问集群)在你的工作站上使用 kubectl 访问集群。在这种情况下，你将通过 Rancher Server 的身份验证代理进行身份验证，然后 Rancher 会让你连接到下游集群。此方法允许你在没有 Rancher UI 的情况下管理集群。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference-guides/cluster-configuration/rancher-server-configuration/use-existing-nodes/use-existing-nodes.md:140]: 身份验证代理进行身份验证 => 认证代理进行认证

File: i18n/zh/docusaurus-plugin-content-docs/current/reference-guides/cluster-configuration/rancher-server-configuration/use-existing-nodes/use-existing-nodes.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+- **通过 kubectl CLI 使用授权的集群端点访问你的集群**：按照[这些步骤](../../../../how-to-guides/new-user-guides/manage-clusters/access-clusters/use-kubectl-and-kubeconfig.md#直接使用下游集群进行身份验证)直接使用 kubectl 访问集群，而无需通过 Rancher 进行身份验证。我们建议设置此替代方法来访问集群，以便在无法连接到 Rancher 时访问集群。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference-guides/cluster-configuration/rancher-server-configuration/use-existing-nodes/use-existing-nodes.md:141]: 进行身份验证 => 进行认证

File: i18n/zh/docusaurus-plugin-content-docs/current/reference-guides/rancher-manager-architecture/rancher-manager-architecture.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+title: 架构
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/zh/reference-guides/rancher-manager-architecture"/>
+</head>
+
+本章节重点介绍 [Rancher Server 及其组件](rancher-server-and-components.md) 以及 [Rancher 如何与下游 Kubernetes 集群通信](communicating-with-downstream-user-clusters.md)
+有关安装 Rancher 的不同方式的信息，请参见[安装选项概述](../../getting-started/overview.md#features-of-the-rancher-api-server)。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference-guides/rancher-manager-architecture/rancher-manager-architecture.md:10]: 这里和下面的概述使用的链接不同，需要检查下是否正确，同时 # 后的标题应替换为对应文件的中文标题

**Review by JacieChao — COMMENTED** [review]
LGTM.
PR will be ready after other dependencies are ready

**Review by martyav — APPROVED** [review]

**Pull request merged**